front/src/components/Chat/clickableMenu.tsx

```diff
@@ -22,6 +22,10 @@ export default function ClickableMenu(props: { user: IUser, chan: IChannel, page
         socket.emit('GiveAdmin', { chanid: props.chan.id, userid: props.user.id });
     }
 
+    const handleUnmute = () => {
+        // socket.emit('Unmute', { chanid: props.chan.id, userid: props.user.id });
+    }
+
```

Comment:
This seems useless for now :grin:

Reply:
ah I know 
i did this in advance before i forget lol
guillaume will do the back very soon so i'll just leave it like this and uncomment it once the back is ready